web-client/src/presenter/computeds/formattedTrialSessionDetailsForFilteredEligibleCases.js

```diff
@@ -0,0 +1,31 @@
+import { formattedTrialSessionDetails } from './formattedTrialSessionDetails';
+import { state } from 'cerebral';
+
+export const formattedTrialSessionDetailsForFilteredEligibleCases = (
+  get,
+  applicationContext,
+) => {
+  const trialSessionDetails = formattedTrialSessionDetails(
+    get,
+    applicationContext,
+  );
+  const filter = get(
+    state.screenMetadata.eligibleCasesFilter.hybridSessionFilter,
+  );
+
+  trialSessionDetails.formattedEligibleCases =
+    trialSessionDetails.formattedEligibleCases.filter(eligibleCase => {
+      const allSmallCases = /^(S+)/;
+      const regularCasesOnly = /^(?!S+)/;
+
+      if (filter === 'Small') {
```

Comment:
🔧 Is there an entity constant that could be used here?

Reply:
Thanks... I did not know there were.  This would have made my life simpler and more productive, since I have not worked much with REGEX.  But it was a good learning experience.